src/TikaOnDotnet.TextExtractor/Extensions.cs

```diff
@@ -1,52 +1,15 @@
-﻿using System;
-using org.apache.tika.config;
-using org.apache.tika.detect;
-using org.apache.tika.io;
-using org.apache.tika.mime;
-
-namespace TikaOnDotNet.TextExtraction
+﻿namespace TikaOnDotNet.TextExtraction
 {
+    /// <summary>
```

Comment:
I am not a fan of doc comments for things that are pretty easy to see from reading the code. Please remove ones like this.